library/src/lib/calendar/calendar.component.ts

```diff
@@ -694,6 +702,7 @@ export class CalendarComponent implements OnInit, OnChanges, AfterViewChecked {
             this.constructCalendar();
             this.constructCalendarYearsList();
         }
+        this.weekDays = this.mondayStartOfWeek ? ['M', 'T', 'W', 'T', 'F', 'S', 'S'] : ['S', 'M', 'T', 'W', 'T', 'F', 'S'];
```

Comment:
do we need to make this translatable?

Reply:
I think so, however I think i18n should be a separate story